lib/main.coffee

```diff
@@ -6,6 +6,12 @@ module.exports =
       type: 'string'
       title: 'Shellcheck Executable Path'
       default: 'shellcheck' # Let OS's $PATH handle the rest
+    userParameters:
+      type: 'string'
+      title: 'Additional Executable Parameters'
+      description:
+        'Additional shellcheck parameters, for example `-x -e SC1090`.'
+      default: ''
```

Comment:
The default for a string is already `''`, no need to specify it unless you are changing that.